cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -178,6 +179,14 @@ rules:
   - list
   - get
   - watch
+- apiGroups:
+  - storage.k8s.io
+  resources:
+  - storageclasses
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
What's going to `watch`? Aren't `get` and `list` sufficient?

Reply:
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/rook/rook_cluster.py#L143
When the orchestrator module fetches any resource using the k8s python client it uses a wrapper class that also watches that resource.

Reply:
What's the purpose of that watch? Is the orchestrator module going to react to events for SC?

Reply:
The way the module uses that watch now is to keep whatever resource it fetches from k8s up to date. Every time there's an event on a resource it updates that resources corresponding object: https://github.com/ceph/ceph/blob/master/src/pybind/mgr/rook/rook_cluster.py#L165